src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -749,6 +749,16 @@ std::optional<winrt::Microsoft::Terminal::Settings::Model::LaunchMode> AppComman
 // - 0 if the commandline was successfully parsed
 int AppCommandlineArgs::ParseArgs(winrt::array_view<const winrt::hstring>& args)
 {
+    for (auto& arg : args)
+    {
+        if (arg == L"-Embedding")

```

Comment:
nit: this one two - should that be `--embedding`? Or is there like a special COM reason it has to be `-Embedding`